draft-ietf-gnap-core-protocol.md

```diff
@@ -1481,7 +1496,7 @@ launching a browser on the end user's device, providing a scannable
 image encoding, and printing out a URI to an interactive
 console. While this URI is generally hosted at the AS, the client
 instance can make no assumptions about its contents, composition,
-or relationship to the AS grant URI.
+or relationship to the grant endpoint URI.
```

Comment:
Ok but we never explicitly define it. Maybe in section 2 we should just add URI in that sentence : "Sending a request to the grant endpoint" so that it's easier to understand

Also we might want to put a note next to the grant definition saying that in GNAP, there's a URI at the AS to manage this.

Reply:
OK, I've added some definitions.